features/step_definitions/take_part_pages_steps.rb

```diff
@@ -34,35 +34,36 @@
   click_on "Update order"
 end
 
-Then(/^I see the take part pages in my specified order including the new page on the frontend get involved section$/) do
-  visit get_involved_path
+Then(/^I see the take part pages in my specified order including the new page on the admin view$/) do
+  visit admin_get_involved_path
+  click_on "Take part pages"
 
-  take_part_headings = page.all(".take-part-pages article h3").map(&:text)
+  # Note that the selector is for the non-JS version of the page
+  take_part_pages = page.all(".well label a").map(&:text)
   @the_take_part_pages_in_order.each.with_index do |take_part_page, idx|
-    expect(take_part_page.title).to eq(take_part_headings[idx])
+    expect(take_part_page.title).to eq(take_part_pages[idx])
   end
 end
 
 When(/^I remove one of the take part pages because it's not something we want to promote$/) do
   visit admin_get_involved_path
   click_on "Take part pages"
 
-  click_on @the_take_part_pages_in_order[0].title
```

Comment:
If `@the_take_part_pages_in_order` is no longer being referenced here, can the other references/definitions be removed too, or do other tests depend on it still?

Reply:
It's still used in the `Then(/^I see the take part pages in my specified order including the new page on the admin view$/) do` block.